build.gradle

```diff
@@ -51,7 +52,7 @@ dependencies {
 		// Reactor Core
 		api "io.projectreactor:reactor-core:$reactorCoreVersion"
 		api "io.projectreactor:reactor-test:$reactorCoreVersion"
-		api "io.projectreactor:reactor-tools:$reactorCoreVersion"
+		api("io.projectreactor:reactor-tools:$reactorCoreVersion") { attributes { attribute(classifiers, ",original") } }
```

Comment:
`{ attributes { attribute(classifiers, ",original") } }` adds the dep without classifier and with classifier "original"
`{ attributes { attribute(classifiers, "original") } }` would only add the dep with classifier "original" no entry without a classifier